Samples/iOS-Swift/iOS-SwiftUITests/SentryFileIOTrackingIntegrationTests.swift

```diff
@@ -0,0 +1,246 @@
+import Foundation
+import Sentry
+import XCTest
+
+//This test is also executed under iOS-SwiftUITests
+//Thats why we need to keep it generic without access
+//to any private part of the sdk
+class SentryFileIOTrackingIntegrationTests: XCTestCase {
+
+    private class Fixture {
+        let data = "SOME DATA".data(using: .utf8) ?? Data()
+        let filePath: String!
+        let fileURL: URL!
+        let fileDirectory: URL!
+        
+        func getOptions(enableAutoPerformanceTracking: Bool = true, enableFileIOTracking: Bool = true, enableSwizzling: Bool = true, tracesSampleRate: NSNumber = 1) -> Options {
+            let result = Options()
+            result.enableAutoPerformanceTracking = enableAutoPerformanceTracking
+            result.enableFileIOTracking = enableFileIOTracking
+            result.enableSwizzling = enableSwizzling
+            result.tracesSampleRate = tracesSampleRate
+            return result
+        }
+        
+        init() {
+            let paths = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask)
+            fileDirectory = paths[0]
+            fileURL = fileDirectory.appendingPathComponent("TestFile")
+            filePath = fileURL?.path
+        }
+    }
+    
+    private var fixture: Fixture!
+    var deleteFileDirectory = false
+    
+    override func setUp() {
+        super.setUp()
+        fixture = Fixture()
+        
+        if  !FileManager.default.fileExists(atPath: fixture.fileDirectory.path) {
+            try? FileManager.default.createDirectory(at: fixture.fileDirectory, withIntermediateDirectories: true, attributes: nil)
+            deleteFileDirectory = true
+        }
+        
+        try? fixture.data.write(to: fixture.fileURL)
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        try? FileManager.default.removeItem(at: fixture.fileURL)
+        if deleteFileDirectory {
+            try? FileManager.default.removeItem(at: fixture.fileDirectory)
+        }
+        clearTestState()
+    }
+    
+    func test_WritingTrackingDisabled_forIOOption() {
+        SentrySDK.start(options: fixture.getOptions(enableFileIOTracking: false))
+        
+        assertSpans {
+            try? fixture.data.write(to: fixture.fileURL)
+        }
```

Comment:
`l`: We could extract this into a function called something like `assertNoSpans`, cause we have it duplicated a couple of times.